.github/workflows/rails-integration-tests.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         name: Compute outputs
         run: |
           rails_versions=$(curl https://rubygems.org/api/v1/versions/rails.json | jq 'group_by(.number[:1])[] | (.[0].number) | select(.[:1]|tonumber > 4)' | jq -s -c)
-          echo "RAILS_VERSIONS=$rails_versions" >> $GITHUB_OUTPUT
+          echo 'RAILS_VERSIONS=["5.2.8.1","6.1.7.8","7.1.3.4"]' >> $GITHUB_OUTPUT
```

Comment:
We probably want to do this on the line above to get the latest non-beta versions:

```
rails_versions=$(curl https://rubygems.org/api/v1/versions/rails.json | jq '[.[] | select(.number | test("beta") | not)] | group_by(.number[:1])[] | (.[0].number) | select(.[:1]|tonumber > 4)' | jq -s -c)
```

This outputs:

```
["5.2.8.1","6.1.7.8","7.1.3.4"]
```

Reply:
Sure, I do prefer the more obvious way in this context (code that is not touched often and so the speed someone can undestand what is going on becomes more important), but let's get this out - updated. 👍